### DISCUSSION TITLE
Shopify - Migrate assets sang store mới với File API và Node.js | Leo's blog

### DISCUSSION THREAD

**Discussion by giscus** [hta218/leohuynh.dev#6] [Announcements]
# Shopify - Migrate assets sang store mới với File API và Node.js | Leo's blog

Sử dụng GraphQL để migrate toàn bộ Assets sang 1 store mới với File API và Node.js

http://localhost:3000/blog/shopify-migrate-files-to-new-store-with-file-api-and-nodejs

**Comment by hta218** [comment]
Thực ra có 1 cách nhanh hơn để sử dụng API với store của bạn là thông qua app **Shopify GraphiQL** - đây là [Official App](https://shopify.dev/apps/tools/graphiql-admin-api) của Shopify để học về GraphQL.

Tuy nhiên vì File API mới được support nên app này vẫn chưa update thêm access scope `read_files` để sử dụng tính đến thời điểm mình viết bài này 😢

![Imgur](https://i.imgur.com/SlMi6BU.png)

Anh em đành chịu khó sử dụng **Postman** với **Private App Access Token** nhé!
